style.css

```
@@ -0,0 +1,104 @@
+@charset "utf-8";
+ @media screen and (min-width:769px){
+    body{
+        font-family: 'Times New Roman', Times, serif;
+    }
+.example{
+    position: relative;
+ }
+
+ .example a{
+    font-weight: bold;
+    font-size: 25px;
+    position:absolute;
+    color: white;
+    top: 50%;
+    left: 50%;
+    -ms-transform: translate(-50%,-50%);
+    -webkit-transform: translate(-50%,-50%);
+    transform: translate(-50%,-50%);
+    padding:0;
+    margin: 0;
+ }
+
+ .example img{
+    width: 100%;
+ }
+
+ .subtitle{
+    text-align: center;
+    color: #C9BCC2;
+    background-color: rgb(223, 208, 0, 0.2);
+ }
+
+ .container{
+    list-style: none;
+    text-align: center;
+    padding: 0.5em 1em;
+    margin: 2em 0;
+    color: black;
+ }
+
+ .photo img{
+    text-align: center;
+    width: 100%;
+ }
+ .footer p{
+    text-align: center;
+    color: #C9BCC2;
+    background-color: rgb(223, 208, 0, 0.2);
+ }
```

idなら`#~`、classなら`.~`、タグなら`~`です

```suggestion
 footer p{
    text-align: center;
    color: #C9BCC2;
    background-color: rgb(223, 208, 0, 0.2);
 }
```